target/linux/generic/hack-5.15/782-usb-net-Marvell_ML352_modem_support.patch

```diff
@@ -0,0 +1,22 @@
+--- a/drivers/usb/serial/option.c
++++ b/drivers/usb/serial/option.c
```

Comment:
Please split this patch into a different commit. And maybe move it to `ramips/patch*` directory. Is there a possibility to upstream it?